torchtune/modules/attention.py

```diff
@@ -10,6 +10,85 @@
 import torch
 from torch import nn
 from torchtune.modules.kv_cache import KVCache
+from torchtune.utils._version import torch_version_ge
+from torchtune.utils.attention_bias import _MaskType
+from torchtune.utils.logging import get_logger, log_once
+
+_log: logging.Logger = get_logger()
+
+
+if torch_version_ge("2.5.0"):
```

Comment:
can we remove the next 70 or so lines after 2.5 is out?

Reply:
once stable version is 2.5, yes. until then we need to keep this check unfortunately. will move it to a separate util